src/containers/MyNdla/Arena/ArenaAdminPage.tsx

```diff
@@ -38,27 +57,27 @@ const ArenaAdminPage = () => {
 
   return (
     <MyNdlaPageWrapper>
-      <HelmetWithTracker title={t("htmlTitles.arenaAdminPage")} />
-      <Heading element="h1" id={SKIP_TO_CONTENT_ID} headingStyle="h1-resource" margin="small">
-        {t("myNdla.arena.admin.title")}
-      </Heading>
-      <Text element="p" textStyle="content-alt">
-        {t("myNdla.arena.admin.description")}
-      </Text>
-      <StyledCardContainer>
-        <AdminNavLink
-          to={"flags"}
-          title={t("myNdla.arena.admin.flags.title")}
-          subText={t("myNdla.arena.admin.flags.description")}
-          icon={<AlertLine />}
-        />
-        <AdminNavLink
-          to={"users"}
-          title={t("myNdla.arena.admin.users.title")}
-          subText={t("myNdla.arena.admin.users.shortDescription")}
-          icon={<UserLine />}
-        />
-      </StyledCardContainer>
+      <PageWrapper>
+        <HelmetWithTracker title={t("htmlTitles.arenaAdminPage")} />
+        <Heading id={SKIP_TO_CONTENT_ID} textStyle="heading.small">
+          {t("myNdla.arena.admin.title")}
+        </Heading>
+        <Text>{t("myNdla.arena.admin.description")}</Text>
+        <StyledCardContainer>
+          <AdminNavLink
+            to={"flags"}
```

Comment:
Kunne vore forenkla litt, men det er veldig pirk
```suggestion
            to="flags"
```